src/use/use.py

```diff
@@ -59,7 +59,6 @@
 import atexit
 import codecs
 import configparser
-import functools
```

Comment:
we'll need functools again, at least for wraps i guess

Reply:
Imported wraps() function, I can add 'import functools' back as well if you like